docs/build_arm.md

```diff
@@ -1,14 +1,14 @@
 # Building Larq Compute Engine for ARM-based systems
-This page descibes how to build a Larq Compute Engine (LCE) inference binary
+This page descibes how to build Larq Compute Engine (LCE) applications
 for 32-bit, as well as 64-bit ARM-based systems. You can either build LCE
 natively on your ARM device or cross-compile it from a non-ARM machine.
 [Bazel](https://bazel.build/) is the primary build system for LCE, and it can
 be used to cross-compile a binary for ARM-based systems using a non-ARM host.
 To natively build on an ARM system itself we provide a script that uses the
 Makefile build system.
 
-This leaves us with three ways to build LCE, which we recommend in the
-following order:
+This leaves us with three ways to build LCE applications, which we recommend in
+the following order:
 1. To cross-compile LCE from a host machine, see the section
```

Comment:
```suggestion
1. To cross-compile LCE from a host machine, see section
```

Reply:
I think in this case we have to leave out the word 'section'. When there's a number, then you leave out the "the", for example "see Section 3".